### PR TITLE
Gherkin Coach module, I delete and add one coach scenario that existing in kolibri

### PR DESCRIPTION
**Pull request by mrpau-julius**
### Summary
Gherkin coach module
I finish all the coach scenarios base on pickle and I delete two .feature that not existing in latest kolibri version. I think that two scenarios was seen in kolibri 0.7.2 and I add one more scenario to complete the Gherkin coach module. 
please review my PR @radinamatic :smiley:

### Contributor Checklist

- [ ] Contributor has fully tested the PR manually
- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

File: integration_testing/features/coach/coach-lesson-report.feature [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Feature: Coach lesson report
+  Coach needs to be able to see the lesson report details
+
+  Background:
+    Given I am signed in to Kolibri as coach user
+      And I am on the *Coach > lessons* page
```
Comment (radinamatic) [inline integration_testing/features/coach/coach-lesson-report.feature]: Capitalize the L in `*Coach > lessons*`.

**Review by radinamatic — CHANGES_REQUESTED** [review]
The two files you deleted were there for a reason:

- `class-coach-view.feature` should describe the scenario that covers the experience of the *Class coach* - coach that has access *only* to the classes they are assigned to, as opposed to the *Facility coach* who has access to *all* the classes in the facility.

- `coach-view-class-report-progress.feature`  should describe the scenario of reviewing the progress of the whole class per each channel. View this video to understand the difference from reviewing the progress of individual learners:
https://drive.google.com/file/d/1bbOUjnBd3K8vvWw4P4gGPbNuRtlwMj18/view

File: integration_testing/features/coach/coach-lesson-report.feature [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      And I see the lesson <lesson_title>
+
+  Scenario: Coach can view the lesson report details
+    When I click the lesson <lesson_title>
+    Then I am on the <lesson_title> page
+      And below I see all the topic <topic>
+    When I click <topic>
+    Then I am on the *lesson report details* page
```
Comment (radinamatic) [inline integration_testing/features/coach/coach-lesson-report.feature]: Capitalize the words in `*lesson report details*` to reflect the UI:

**Comment by mrpau-julius** [comment]
thank you so much @radinamatic for review my PR, I thoughts class coach and facility coach are same access but not. yes your right that two coach users are different. I will edit that today. so that I can proceed to guest user :smiley:

**Review by radinamatic — APPROVED** [review]
Looking better @mrpau-julius! 👍

**Pull request merged**